File: pf-core/src/main/java/com/purdynet/util/DateUtil.java
package com.purdynet.util;

import java.util.Calendar;
import java.util.Date;

/**
 * Created with IntelliJ IDEA.
 * User: dnpurdy
 * Date: 9/2/13
 * Time: 10:34 PM
 * To change this template use File | Settings | File Templates.
 */
public class DateUtil
{
    public static Date getDate(int year, int month, int day)
    {
        Calendar c = Calendar.getInstance();
        c.set(year, month-1, day);
        return c.getTime();
    }

    public static Integer getYear(Date date)
    {
        Calendar c = Calendar.getInstance();
        c.setTime(date);
        return c.get(Calendar.YEAR) - 1900;
    }
}

File: pf-core/src/main/scala/com/purdynet/util/DateUtil.scala
package com.purdynet.util

import java.util.{Calendar, Date}

/**
  * Created by dnpurdy on 11/2/16.
  */
object DateUtil {
  def getDate(year: Int, month: Int, day: Int): Date = {
    val c: Calendar = Calendar.getInstance
    c.set(year, month - 1, day)
    c.getTime
  }
}
